personalize.admin.inc

```diff
@@ -205,7 +205,7 @@ function personalize_admin_build_visitor_context_select($settings, $exclude_disa
   }
   $element = array(
     '#type' => 'select',
-    '#title' => t('Visitor contexts to use for automatic targeting'),
+    '#title' => t('Visitor contexts'),
```

Comment:
The disabled visitor context stuff still uses it and I figured there is still value to have this in personalize module so I left it in there... but made the text more generic.